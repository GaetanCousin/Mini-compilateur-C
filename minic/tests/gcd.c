extern int printf();
extern void exit(int x);
int main() {
            int a;
            int b;
            int x;
            int y;
            a = 2000;
            b = 2000;
            x = a;
            y = b;
      for (;b != 0;)
      {
              int t;
              t = b;
              b = a % b;
              a = t;
      }
      printf("Le pgcd de %d et %d est %d\n", x, y, a);

      exit(0);
      return 0;
}
