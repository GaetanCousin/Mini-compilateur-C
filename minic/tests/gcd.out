Le pgcd de 2000 et 2000 est 2000
